### DISCUSSION TITLE
Windows -path does not read files

### DISCUSSION THREAD

**Discussion by Erenlancaster** [projectdiscovery/httpx#876] [Q&A]
### httpx version:

1.2.4
### Current Behavior:

-path filename does not work on windows
### Expected Behavior:

the application works properly on windows
### Steps To Reproduce:

http.exe -l hosts.txt -path path.txt -sc -v

### Anything else:

![image](https://user-images.githubusercontent.com/8293321/191001234-8ec3ec71-fc46-4987-9b44-8c58283d30fe.png)

**Comment by Mzack9999** [comment]
The tools seem to behave as expected:
```go
PS C:\Users\user\go\src\github.com\projectdiscovery\httpx\cmd\httpx> cat .\hosts.txt
192.168.1.1
PS C:\Users\user\go\src\github.com\projectdiscovery\httpx\cmd\httpx> cat .\path.txt
/test1
/test2
PS C:\Users\user\go\src\github.com\projectdiscovery\httpx\cmd\httpx> .\httpx.exe -l hosts.txt -path path.txt -sc -v

    __    __  __       _  __
   / /_  / /_/ /_____ | |/ /
  / __ \/ __/ __/ __ \|   /
 / / / / /_/ /_/ /_/ /   |
/_/ /_/\__/\__/ .___/_/|_|
             /_/              v1.2.4

                projectdiscovery.io

Use with caution. You are responsible for your actions.
Developers assume no liability and are not responsible for any misuse or damage.
https://192.168.1.1/test1 [404]
https://192.168.1.1/test2 [404]
```
Could you ensure that the file `path.txt` exists in the current folder?

**Comment by Erenlancaster** [comment]
![image](https://user-images.githubusercontent.com/50498704/193332354-44f89f4c-742c-41f6-b32f-3bb4c282b1b6.png)

i still get the same error all files are in the folder

**Comment by Mzack9999** [comment]
This behavior is really weird. I tried to download the [httpx binary](https://github.com/projectdiscovery/httpx/releases/download/v1.2.4/httpx_1.2.4_windows_amd64.zip) directly and place it in the onedrive folder along with the files `hosts` and `path.txt` files:
![image](https://user-images.githubusercontent.com/13421144/193597027-64e5c59b-87cd-4d94-b8d0-936292e966f9.png)

If you execute the command in powershell or using `.\path.txt` do you still get the same result?

**Comment by ehsandeep** [comment]
Moving to the discussion as we are unable to replicate this and require more information before we can work on this.
